corehq/util/decorators.py

```diff
@@ -102,7 +102,7 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         pass
 
 
-class CouldNotAqcuireLock(Exception):
+class CouldNotAcquireLock(Exception):
     pass
```

Comment:
I suppose we could add a comment here mentioning that this used to be name `CouldNotAqcuireLock` in case useful for someone looking for more history in sentry? Not sure how useful that would be.